DataFormats/TrackReco/src/classes_def.xml

```diff
@@ -1,5 +1,39 @@
 <lcgdict>
 
+  <class name="reco::storage::TrackMomentumValues" ClassVersion="3">
+    <version ClassVersion="3" checksum="3766989023"/>
+  </class>
+  <class name="reco::storage::TrackMomentumStorage" ClassVersion="3">
+      <version ClassVersion="3" checksum="4153617590"/>
+  </class>
+  <ioread
+   sourceClass="ROOT::Math::DisplacementVector3D<ROOT::Math::Cartesian3D<Double32_t>,ROOT::Math::DefaultCoordinateSystemTag>"
+    targetClass="reco::storage::TrackMomentumStorage"
+    version="[1-]"
```

Comment:
what's the effect of this; does it mean that any `math::XYZVector` will be mapped to `reco::storage::TrackMomentumStorage` even outside of `reco::Track`?

Reply:
That is not what it means. The source is what is in the file so this would be for an older version of the class which used the Vector. The targetClass is the type within the data product in memory. 

What causes the Vector class to be written out as the new type is the explicit declaration later in the file of
```
   <field name="vertex_" iotype="reco::storage::TrackPositionStorage" /> 
    <field name="momentum_" iotype="reco::storage::TrackMomentumStorage" /> 
```
as that is part of `reco::TrackBase` declaration it will only happen to that class.